app/Http/Controllers/TaskCategoriesController.php

```diff
@@ -22,7 +22,10 @@ public function index(Request $request)
     {
         $categories = $request->user()->taskCategories;
 
-        return view('task-category.index', compact('categories'));
+        TaskCategoryResource::withoutWrapping();
+        return new TaskCategoryResource($categories);
```

Comment:
For returning a collection you need to use `return TaskCategoryResource::collection($categories)`

Reply:
ok sir.